packages/cf.js/API_REFERENCE.md

```diff
@@ -427,8 +427,6 @@ An instance of an installed app.
     - Amount of the asset deposited by the counterparty
 - `timeout: BigNumber`
     - Number of blocks until a submitted state for this app is considered finalized
-- `isVirtual: boolean`
```

Comment:
Adding this boolean would break a bunch of Node code, which I thought was unecessary since you can derive it from the `intermediaries` field.